schemas/core/modes/MODE_CAR.json

```diff
@@ -1,4 +1,5 @@
 {
+  "$id": "http://maasglobal.com/core/modes/MODE_CAR.json",
   "description": "Schemas for MODE_CAR meta field",
-  "$ref": "../components/car-rental.json"
+  "$ref": "http://maasglobal.com/core/components/car-rental.json"
```

Comment:
it's relative to the id of the parent schema actually. So if this is `/core/modes/MODE_CAR.json`
Without absolute URI it would be resolve incorrectly ( path.resolve is how ajv use to resolve reference)

Reply:
A bit weird, as it's urls so it should rather use [url.resolve](https://nodejs.org/api/url.html#url_url_resolve_from_to) :)

Anyway if it's indeed `path.resolve` it needs to stay as it is